include/ginkgo/extensions/config/json_config.hpp

```diff
@@ -80,6 +80,13 @@ inline gko::config::pnode parse_json_file(std::string filename)
     return parse_json(nlohmann::json::parse(fstream));
 }
 
+/**
+ * parse_json_string takes a json string to generate the property tree object
+ */
+inline gko::config::pnode parse_json_string(std::string json)
```

Comment:
I wonder if it's a good time to start introducing `std::string_view` to the code base, now that we support C++17